src/layouts/mainPageSections/welcomeSection.js

```diff
@@ -60,11 +60,11 @@ const WelcomeSection = () => (
       <SectionHeading>Łączy nas PRZYSZŁOŚĆ</SectionHeading>
       <TextParagraph>
         Jeśli dbasz o siebie, swoją rodzinę, majątek lub firmę - znalazłeś się w
-        dobrym miejscu. Wielopokoleniowi to blog sukcesji, mądrym budowaniu
+        dobrym miejscu. <i>Wielopokoleniowi<i/>to blog sukcesji, mądrym budowaniu

```

Comment:
Tagi zamyka się w drugą stronę
```suggestion
        dobrym miejscu. <i>Wielopokoleniowi</i>to blog sukcesji, mądrym budowaniu

```